### PR TITLE
Don't run phpunit tests on Travis

### PR DESCRIPTION
**Pull request by teohhanhui**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | N/A
| License       | MIT
| Doc PR        | N/A

Since they're already run in the CI workflow on GitHub Actions

TODO:

- [x] Move the rest of the phpunit tests to GitHub Actions too, perhaps?

### CODE REVIEW

**Comment by teohhanhui** [comment]
Jobs with `7.4snapshot` on Travis are failing. Not sure if it's a problem with their build of PHP 7.4.

File: .travis.yml [hunk 1]
```diff
@@ -49,68 +45,93 @@ jobs:
...
-        - *run-phpunit-tests
+        - *run-behat-tests
+        - *validate-openapi-v2-json
+        - *validate-openapi-v2-yaml
+        - *validate-openapi-v3-json
+        - *validate-openapi-v3-yaml
+
+    - name: 'Behat (PHP 7.4-rc) (lowest dependencies)'
+      php: '7.4snapshot'
+      env: LEGACY=0
```
Comment (teohhanhui) [inline .travis.yml]: @dunglas Is this okay? At least until https://github.com/michelf/php-markdown/pull/316 is released?

**Comment by teohhanhui** [comment]
We need to wait for the next Twig 2.x release: https://github.com/twigphp/Twig/issues/3151

**Comment by teohhanhui** [comment]
Other failures should have been fixed. Waiting for next Twig 2.x release...

**Comment by teohhanhui** [comment]
Still targeting `2.4` as this does not change any code, only the testing / CI parts.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -72,27 +67,70 @@ jobs:
...
         if: matrix.php != '7.1'
         run: |
+          apk add $PHPIZE_DEPS
           pecl install mongodb-1.5.5
           docker-php-ext-enable mongodb
       - name: Disable PHP memory limit
         run: echo 'memory_limit=-1' >> /usr/local/etc/php/php.ini
       - name: Install Composer
-        run: curl -sS -o - https://raw.githubusercontent.com/composer/getcomposer.org/ba13e3fc70f1c66250d1ea7ea4911d593aa1dba5/web/installer | php -- --install-dir=/usr/local/bin --filename=composer --quiet
+        run: wget -qO - https://raw.githubusercontent.com/composer/getcomposer.org/fb22b78362d31c0d2bf516d1f8cdfd2745caa431/web/installer | php -- --install-dir=/usr/local/bin --filename=composer --quiet
```
Comment (soyuka) [inline .github/workflows/ci.yml]: can't we use a variable for this composer sha? may ease maintenance?

Comment (teohhanhui) [reply .github/workflows/ci.yml]: GitHub workflow files don't support YAML anchors yet... :disappointed:

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -72,27 +67,70 @@ jobs:
...
       - name: Clear test app cache
         run: tests/Fixtures/app/console cache:clear --ansi
       - name: Run PHPUnit tests
         run: vendor/bin/phpunit --colors=always
+
+  phpunit-no-deprecations:
+    name: PHPUnit (PHP ${{ matrix.php }}) (no deprecations)
+    runs-on: ubuntu-latest
+    container:
+      image: php:${{ matrix.php }}-alpine
```
Comment (bendavies) [inline .github/workflows/ci.yml]: any reason to use a docker image here rather than php installed on  `ubuntu-latest`?

Comment (bendavies) [reply .github/workflows/ci.yml]: ah, you are using `7.4-rc` elsewhere which isn't available. ok

Comment (teohhanhui) [reply .github/workflows/ci.yml]: Because we want full control over the PHP version being used. :smile:

Comment (bendavies) [reply .github/workflows/ci.yml]: ok, but how does using `php:7.3-alpine` give you full control? you need to fix the the patch release for pull control?

Comment (teohhanhui) [reply .github/workflows/ci.yml]: > how does using php:7.3-alpine give you more control over php7.3 on ubuntu-latest?

For example, there's no `php7.4-rc`. Anyway, it's good to be CI provider-agnostic as much as possible.

> you need to fix the the patch release for full control?

We will do that when necessary. Using the preinstalled version in the VM prevents that.

Comment (bendavies) [reply .github/workflows/ci.yml]: 👍

**Comment by teohhanhui** [comment]
Healthcheck for `elasticsearch` container finally fixed lol

**Pull request merged**